### ISSUE TITLE
Still getting some spam restrooms

### ISSUE THREAD

**Issue by tkwidmer** [RefugeRestrooms/refugerestrooms#217]
I found a couple on prod today.
@backspace, it looks like Akismet may not be taking care of all cases.

**Comment by backspace** [comment]
Yeah, no anti-spam solution will be perfect. There can be false positives and false negatives. The Akismet API has the [Submit Spam](http://akismet.com/development/api/#submit-spam) endpoint for submitting false negatives. It could be connected to whatever admin review features exist.

**Comment by tkwidmer** [comment]
That would be cool.

**Comment by kiesia** [comment]
:+1:
